spec/concurrent/executor/java_single_thread_executor_spec.rb

```diff
@@ -4,7 +4,7 @@
 
   module Concurrent
 
-    describe JavaSingleThreadExecutor, :type=>:jruby do
+    describe JavaSingleThreadExecutor, type::jruby do
```

Comment:
This line needs a space between the two colons (`type: :jruby`). It's raising a ``<module:Concurrent>': undefined local variable or method `type' for Concurrent:Module (NameError)` exception on all platforms. Please update the PR.